website/docs/d/resource_schema.html.markdown

```diff
@@ -0,0 +1,360 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_resource_schema"
+sidebar_current: "docs-vcd-datasource-resource-schema"
+description: |-
+  Provides information about a vCD resource structure
+---
+
+# vcd\_resource_schema
+
+Provides a vCloud Director generic structure data source. It shows the structure of any VCD resource.
+
+Supported in provider *v3.1+*
+
+## Example Usage 1
+
+Showing a structure with simple attributes only 
+
+```hcl
+data "vcd_resource_schema" "org_struct" {
+  name          = "org_struct"
+  resource_type = "vcd_org"
+}
+
+// Shows the organization attributes
+output "org_struct" {
```

Comment:
Something is wrong with the computed fields. Try running the `apply` for the second time with exactly the same configuration and you'll end up with many changes:
![Screenshot 2020-11-25 at 14 04 17](https://user-images.githubusercontent.com/11330689/100225783-6d5d6780-2f27-11eb-82d8-d3194b405427.png)

Reply:
Changed the representation from list to set. It now ignores listing order